### PR TITLE
Refs [TASK][34374] Show city in screen explore receive data to api

### PR DESCRIPTION
**Pull request by phuongnam12a4gt**
## Related Tickets
- [#Task 34374: load Data City]
- https://edu-redmine.sun-asterisk.vn/issues/34374
## 
Nhận dữ liệu của thành phố về tên của thành phố.
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>

### CODE REVIEW

File: app/src/main/java/com/sun/americanroom/screen/explore/ExploreRoomFragment.kt [hunk 1]
```diff
@@ -37,9 +52,23 @@ class ExploreRoomFragment : Fragment(),
...
     }
 
+    private fun initData() {
+        explorePresenter.run {
+            setView(this@ExploreRoomFragment)
+            getCityExplore(StateCode.CALIFORNIA)
+        }
+    }
+
     override fun onItemClickListener(item: City?) {
```
Comment (hoangphuc612) [inline app/src/main/java/com/sun/americanroom/screen/explore/ExploreRoomFragment.kt]:
```suggestion
    override fun onItemClickListener(item: City?) {}
```

File: app/src/main/java/com/sun/americanroom/screen/explore/ExploreRoomFragment.kt [hunk 2]
```diff
@@ -37,9 +52,23 @@ class ExploreRoomFragment : Fragment(),
...
     }
 
+    override fun getCityOnSuccess(exploreCity: MutableList<City>) {
+        adapterCityExplore.updateData(exploreCity)
+    }
+
+    override fun onError(exception: Exception?) {
```
Comment (hoangphuc612) [inline app/src/main/java/com/sun/americanroom/screen/explore/ExploreRoomFragment.kt]:
```suggestion
    override fun onError(exception: Exception?) {}
```

File: app/src/main/java/com/sun/americanroom/screen/explore/ExploreRoomFragment.kt (hunk 1 above)
Comment (phuongnam12a4gt) [reply app/src/main/java/com/sun/americanroom/screen/explore/ExploreRoomFragment.kt]: Cảm ơn! mình đả sửa

File: app/src/main/java/com/sun/americanroom/screen/explore/ExploreRoomFragment.kt (hunk 2 above)
Comment (phuongnam12a4gt) [reply app/src/main/java/com/sun/americanroom/screen/explore/ExploreRoomFragment.kt]: Cảm ơn! Mình đả sửa.

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**